plugins/http_plugin/http_plugin.cpp

```diff
@@ -67,10 +67,11 @@ class http_plugin_impl : public std::enable_shared_from_this<http_plugin_impl> {
           * @param my - the http_plugin_impl
           * @return the constructed internal_url_handler
           */
-         static detail::internal_url_handler make_app_thread_url_handler( int priority, url_handler next, http_plugin_impl_ptr my ) {
+         static detail::internal_url_handler make_app_thread_url_handler( int priority, appbase::exec_queue q, url_handler next, http_plugin_impl_ptr my ) {
             auto next_ptr = std::make_shared<url_handler>(std::move(next));
-            return [my=std::move(my), priority, next_ptr=std::move(next_ptr)]
+            return [my=std::move(my), priority, &q, next_ptr=std::move(next_ptr)]
```

Comment:
q is a stack variable, so it shouldn't be captured by reference

Reply:
A good catch, thanks! & was left from an earlier version.